site/sfguides/src/liveramp_identity_and_translation_quickstart/liveramp_identity_and_translation_quickstart.md

```diff
@@ -0,0 +1,448 @@
+id: liveramp_identity_and_translation_quickstart
+summary: Identity Resolution and Transcoding with LiveRamp and Snowflake
+categories: Marketing,Getting-Started,Partner-Integrations
+environments: web
+status: Published 
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Getting Started, Data Science, Data Engineering, MarTech, Marketing
+
+# Identity Resolution and Transcoding with LiveRamp and Snowflake
```

Comment:
@ShawnGilleran @sfc-gh-lambrosetti Missing "What You'll Need" section with at least the need to have a Snowflake account?

Reply:
@iamontheinet I have pushed a new commit to resolve the items